website/pages/docs/quickstart.mdx

```diff
@@ -42,6 +42,8 @@ Million.js is a library that enables you to create _blocks_. A block is a specia
 Blocks are essentially components wrapped by `block(){:jsx}`.
 
 ```jsx {7}
```

Comment:
nice, but you'll need to update the highlighted line. in this case, it would be {9}

Reply:
do this for every instance you've updated